waku/v2/protocol/waku_relay.nim

```diff
@@ -78,15 +77,12 @@ method initPubSub(w: WakuRelay) {.raises: [InitializationError].} =
   w.initProtocolHandler()
 
 
-proc new*(T: type WakuRelay,
-          switch: Switch,
-          defaultPubsubTopics: seq[PubsubTopic] = @[],
-          triggerSelf: bool = true): WakuRelayResult[T] =
+proc new*(T: type WakuRelay, switch: Switch, triggerSelf: bool = true): WakuRelayResult[T] =
 
   proc msgIdProvider(msg: messages.Message): Result[MessageID, ValidationResult] =
     let hash = MultiHash.digest("sha2-256", msg.data)
     if hash.isErr():
-      ok(($msg.data.hash).toBytes())
+      ok(toBytes($hashes.hash(msg.data)))
```

Comment:
I have no idea how the `waku_relay` module was compiled 🤯  

The `std/hashes` import was missing for this line. It took me 2 days to find the right import that fixed this line's compilation error.